api/harborapi_test.go

```diff
@@ -137,7 +149,49 @@ func (a api) ProjectsPost(prjUsr usrInfo, project apilib.ProjectReq) (int, error
 
 	// body params
 	_sling = _sling.BodyJSON(project)
-	httpStatusCode, _, err := request(_sling, jsonAcceptHeader, prjUsr)
+	_, httpStatusCode, _, err := request(_sling, jsonAcceptHeader, prjUsr)
+	return httpStatusCode, err
+
+}
+
+//User Add
+func (a api) UserAdd(user apilib.User, authInfo ...usrInfo) (int, int, error) {
+
+	_sling := sling.New().Post(a.basePath)
+
+	// create path and map variables
+	path := "/api/users/"
+
+	_sling = _sling.Path(path)
+
+	// body params
+	_sling = _sling.BodyJSON(user)
+	var httpStatusCode, userID int
+	var err error
+	if len(authInfo) > 0 {
+		userID, httpStatusCode, _, err = request(_sling, jsonAcceptHeader, authInfo[0])
+	} else {
+		userID, httpStatusCode, _, err = request(_sling, jsonAcceptHeader)
+	}
+	return userID, httpStatusCode, err
+}
+
+//User Delete
+func (a api) UserDelete(userID int, authInfo ...usrInfo) (int, error) {
+
+	_sling := sling.New().Delete(a.basePath)
+
+	// create path and map variables
+	path := "/api/users/" + fmt.Sprintf("%v", userID)
+	_sling = _sling.Path(path)
+
+	var httpStatusCode int
+	var err error
+	if len(authInfo) > 0 {
```

Comment:
why use if here? not better why.
please read project delete code.
---------------

func (a api) ProjectsDelete(prjUsr usrInfo, projectID string) (int, error) {
	_sling := sling.New().Delete(a.basePath)

	//create api path
	path := "api/projects/" + projectID
	_sling = _sling.Path(path)
	_, httpStatusCode, _, err := request(_sling, jsonAcceptHeader, prjUsr)
	return httpStatusCode, err
}
--------